### DISCUSSION TITLE
Выбранное в новой строке

### DISCUSSION THREAD

**Discussion by Makhmut2445** [totumonline/totum-ru-issues-and-discussions#166] [Как это сделать?] [answered]
Есть поле в котором я выбираю значения через селект дерево.
Как сделать чтоб после выбора, в другой таблице создались новые строки записав эти выбранные значения каждую в новую строку?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
`Дерево` — это `Селект`, а `мультидерево` как и `мультиселект` хранит `list`.

Как определить что добавлено — по разности листов — `listMinus`.

Что бы определить что:

+ Добавлено — надо из **текущего** листа вычесть **предыдущий**
+ Удалено — надо из **предыдущего** листа вычесть **текущий**

Что потом — `listInsert`

В виде кода-действия на добавление/изменение/удаление:

```
a1=: insertList(table: 'some_table'; field: 'some_field' = $listAdd; log: true)

	listAdd: listMinus(list: $#nfv; list: $#onfv)

a2=: deleteList(table: 'some_table'; where: 'some_field' = $listDel; log: true)

	listDel: listMinus(list: $#onfv; list: $#nfv)
```
